doc/whatsnew/fragments/7214.bugfix

```diff
@@ -0,0 +1,3 @@
+Message send to reporter are now copied so a reporter cannot modify the message sent to other reporters.
+
+Refs #7214
```

Comment:
should this be `Closes`?

Reply:
I wasn't sure of semantics of this, but yes ! Let me fix that.